cleverhans/model.py

```diff
@@ -101,7 +102,6 @@ def __init__(self, callable_fn, output_layer):
         :param output_layer: A string of the output layer returned by the
                              function. (Usually either "probs" or "logits".)
         """
-
```

Comment:
This change is unrelated to eager.

Reply:
True, updated PR will reflect the change.